apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-tabs.tsx

```diff
@@ -59,6 +63,7 @@ export const InAppTabs = ({ workflow, step }: { workflow: WorkflowResponseDto; s
   const { isPending, updateWorkflow } = useUpdateWorkflow({
     onSuccess: (data) => {
       resetWorkflowForm(data);
+      refetchStep();
```

Comment:
When the new form values are saved, we execute `PUT` request to save the workflow (incl. new control values), but the response (updated workflow) doesn't contain control values in DTO - we need to call refetch on step to get the newly updated control values, so the form updates.

I'm not sure if I like this approach, maybe we should include the control values in workflow DTO and this logic wouldn't be needed.

Reply:
This is fine for now, we will later use the `PATCH /workflows/:id/steps/:id` endpoint that should solve this problem.